### PR TITLE
Alice revokes

### LINKED ISSUE #10: Revoke blockchain policy
* Currently there is no way for Alice to revoke a blockchain policy once it was granted to Bob
* Implement revocation mechanism

### PR DESCRIPTION
**Pull request by piotr-roslaniec**
- Implements only on-chain part of the revocation flow
- Contains minor changes to how blockchain policy parameters are calculated and validated
- Fixes gas limit for policy creation being too low for multiple nodes
- Removes some unused code (arrangements, off-chain revocation)
- Closes #10

### CODE REVIEW

File: src/agents/policy-manager.ts [hunk 1]
```diff
@@ -44,6 +45,33 @@ export class PolicyManagerAgent {
     return tx;
   }
 
+  public static async policyDisabled(
+    transactingPower: TransactingPower,
```
Comment (KPrasch) [inline src/agents/policy-manager.ts:49]: Considering that this is a read-only function, why is `transactingPower` needed here?

File: src/characters/alice.ts [hunk 2]
```diff
@@ -185,4 +187,51 @@ export class Alice {
       policyEndTime
     ) as BlockchainPolicyParameters;
   }
+
+  public async revoke(
+    policy: EnactedPolicy
```
Comment (KPrasch) [inline src/characters/alice.ts:192]: Considering that only `policyID` is used from `EnactedPolicy`, and alice may be revoking this policy without having any good way to instantiate an EnactedPolicy in a separate runtime form the granting operation, perhaps we just accept the policy ID and not an entire Policy here.

Comment (piotr-roslaniec) [reply src/characters/alice.ts:192]: But don't we need other things such as revocation kits for off-chain revocation? This is not exactly a priority in this issue

Comment (piotr-roslaniec) [reply src/characters/alice.ts:189]: How about:
```
 public async revoke(
    // provide atleast one:
    policyId?: Uint8Array, // to revoke on-chain 
    revocationKit?: RevocationKit // to revoke off-chain
  ) 
```

**Comment by piotr-roslaniec** [comment]
@derekpierre That's true, this is a left-over code. Since we are only working with on-chain revocation we can also remove other related lines.

**Pull request merged**